jetpack-compose-article-impl/app/src/main/java/com/fastlink/jetpack_compose_article_impl/HomeScreen.kt

```diff
@@ -0,0 +1,90 @@
+package com.fastlink.jetpack_compose_article_impl
+
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.animation.animateContentSize
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+
+
+@Composable
+fun HomeScreen(modifier: Modifier = Modifier){
+    LazyColumn(modifier = modifier
+        .fillMaxSize()
+        .background(MaterialTheme.colorScheme.background)
+        .padding(horizontal = 10.dp,)) {
+       items(conversationSample){ message ->
+                MessageCard(message)
+            }
+    }
+}
+
+
+@Composable
+fun MessageCard(message: Message){
+    Row (modifier = Modifier
+        .padding(10.dp)){
+        Image(painter = painterResource(id =message.image),
+            contentDescription = null,
+            modifier= Modifier
+                .size(50.dp)
+                .clip(CircleShape)
+                .border(1.5.dp, MaterialTheme.colorScheme.primary, CircleShape),
+            contentScale = ContentScale.FillBounds,
+        )
+         var isExpanded by  rememberSaveable{
```

Comment:
The variables usually should be on top, that makes the function easy to read and understand.